src/nunit.gui.core/Model/TestModel.cs

```diff
@@ -202,12 +205,12 @@ public void NewProject()
 
         public void NewProject(string filename)
         {
-            MessageBox.Show("It's not yet decided if we will implement saving of projects. The alternative is to require use of the project editor, which already supports this.", "Not Yet Implemented");
+            _messageDisplay.Info("It's not yet decided if we will implement saving of projects. The alternative is to require use of the project editor, which already supports this.", "Not Yet Implemented");
```

Comment:
yeah sorry about that.. I'll have to draw up a diagram and think about what i'm doing when i'm playing with the models and views and presenters. I've still not got the pattern fully committed to memory.